templates/_layout.html

```diff
@@ -61,6 +61,10 @@
           {
             'text': 'Opt out of emergency alerts',
             'href': '/alerts/opt-out'
+          },
+          {
+            'text': 'Previous alerts',
+            'href': '/alerts/previous-alerts'
           }
```

Comment:
We should include the link to current alerts because the two are complementary, even though the page will be empty for now:
```suggestion
          },
          {
            'text': 'Current alerts',
            'href': '/alerts/current-alerts'
          },
          {
            'text': 'Previous alerts',
            'href': '/alerts/previous-alerts'
          }
```

Reply:
Updated - I've put them as the first two links, as in the prototype.